packages/compass/src/setup-hadron-distribution.ts

```diff
@@ -61,3 +61,38 @@ if (
     path.join(app.getPath('userData'), 'CrashReporter')
   );
 }
+
+if (
+  // type `renderer` is electron renderer process (browser window runtime)
+  process.type === 'renderer'
+) {
+  if (process.env.NODE_ENV === 'development') {
+    const ignoreLeafygreenWarnings = [
+      // Not relevant
+      /using the Leafygreen SearchInput/i,
+      // We don't always use SegmentedControl as a view switcher, aria-controls
+      // doesn't apply
+      /The property `aria-controls` is required/i,
+      // TODO(COMPASS-7046): Should go away after leafygreen update
+      /For screen-reader accessibility, label or aria-labelledby/i,
```

Comment:
I don't feel too great about doing this (will ask leafygreen if they'd be open to disabling the one for SegmentedControl, and the other should go away) but they are very noisy when working in the application locally, so I'd prefer to disable them

Please say though if you feel like we shouldn't do that